auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -137,4 +140,9 @@ AuthenticationRequest createAuthenticationRequest(HttpUrl url, OkHttpClient clie
     Map<String, String> getHeaders() {
         return headers;
     }
+
+    static String getDefaultLocale() {
+        String language = Locale.getDefault().toString();
+        return !language.isEmpty() ? language : "en";
```

Comment:
I took it from here https://gist.github.com/jasef/337431c43c3addb2cbd5eb215b376179 but I guess I can add the country. Defaulting to US seems fine, anyway this is an edge case. The phone should have a valid locale.